From 42b49f7945f73d29485fcd180641104af56d031f Mon Sep 17 00:00:00 2001
From: Tim Kordas <tim.kordas@joyent.com>
Date: Mon, 20 Nov 2017 23:06:02 +0000
Subject: [PATCH] MANTA-3364 'Loadbalancer has no backends' probe is missing
 tooling to actually run Reviewed by: David Pacheco <dap@joyent.com>

---
 package.json | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package.json b/package.json
index c95330f..da15204 100644
--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
         "dashdash": "1.14.1",
         "forkexec": "1.1.0",
         "jsprim": "1.4.0",
+        "haproxy-stat" : "0.1.0",
         "node-uuid": "1.4.1",
         "once": "1.4.0",
         "vasync": "2.0.0",
-- 
2.21.0

